app/javascript/Components/Result/rubric_criterion_input.jsx

```diff
@@ -25,13 +27,24 @@ export default function RubricCriterionInput({
   };
 
   // The parameter `level` is the level object selected
-  const renderRubricLevel = level => {
+  const renderRubricLevel = (level, index) => {
     const levelMark = level.mark.toFixed(2);
     let selectedClass = "";
     let oldMarkClass = "";
+    let activeRubricClass = "";
     if (mark !== undefined && mark !== null && levelMark === mark.toFixed(2)) {
       selectedClass = "selected";
     }
+    if (active) {
+      // If this level is selected, make it active
```

Comment:
Since the two branches have the same effect, you can combine the conditions together:

```
if (selectedClass === "selected" || (...))
```

Write one comment that describes both of these cases